### ISSUE TITLE
AbcImport support IPoints?

### ISSUE THREAD

**Issue by caron** [alembic/alembic#94]
Is there any plan by the Alembic maintainers or Autodesk devs to support importing of points/particles?

**Comment by lamiller0** [comment]
Good question.
It's something that I've wanted to do, but hasn't had a high priority.
Historically MFnParticleSystem wasn't suitable for the types of point data we could get in other packages, and not everyone had access to nParticles.
If you have something already implemented, feel free to submit a pull request..

**Comment by caron** [comment]
I don't have anything actually, I also have very little experience with the Maya API so I couldn't offer it either. I am saddened Autodesk hasn't extended it themselves?

**Comment by HansBaldzuhn** [comment]
I submitted a pull request for this particular feature. I'm waiting for review and advices !

**Comment by BigRoy** [comment]
Does #163 being merged with #202 mean this issue is fixed? Or are particles in Maya now working as expected to import and export? And if so, I'm expecting they can also import Houdini particles as expected - correct?

**Comment by lamiller0** [comment]
Correct, support for particles has been added.  The goal is that Houdini particles are also be supported.  We'll see how well it pans out.
